setup.py

```diff
@@ -21,8 +21,8 @@
     use_scm_version=True,
     url='https://github.com/labgrid-project',
     data_files=[('share/man/1', ['man/labgrid-client.1',
-                                 'man/labgrid-exporter.1',
-                                 'man/labgrid-device-config.1'])],
+                                 'man/labgrid-exporter.1']),
+                ('share/man/5', ['man/labgrid-device-config.5'])],
```

Comment:
I think `data_files` is missing entries for `labgrid-pytest.7` and `labgrid-suggest.1`

Reply:
Added.